packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,130 @@
 <template>
-  <va-sidebar :minimized="minimized">
+  <va-sidebar
+    class="sidebar"
+    color="secondary"
+  >
     <template slot="menu">
-      <template v-for="(item, key) in items">
-        <va-sidebar-link-group
-          v-if="item.children"
-          :key="key"
-          :minimized="minimized"
-          :title="$t(item.displayName)"
-          :children="item.children"
-        >
-          <va-sidebar-link
-            v-for="(subMenuItem, index) in item.children"
-            :key="index"
-            :to="subMenuItem.name"
-            :title="$t(subMenuItem.displayName)"
-          />
-        </va-sidebar-link-group>
-        <va-sidebar-link
-          v-else
-          :key="key"
-          :minimized="minimized"
-          :active-by-default="item.name === route.name"
-          :to="item.name"
-        >
-          <span slot="title">{{ $t(item.displayName) }}</span>
-        </va-sidebar-link>
-      </template>
+      <va-list class="sidebar__links">
+        <va-expand-group v-model="value">
+          <va-expand
+            v-for="(item, key) in items"
+            :key="key"
+            class="sidebar__expand"
+            :class="{'sidebar__expand--active': value[key]}"
+          >
+            <template slot="header">
+              <va-list-item class="sidebar__category">
+                <va-list-item-section class="sidebar__category__section">
+                  <va-list-item-label class="sidebar__category__label">
+                    {{ $t(item.displayName) }}
+                  </va-list-item-label>
+                </va-list-item-section>
+                <va-list-item-section icon>
+                  <va-icon
+                    :name="value[key] ? 'expand_less' : 'expand_more'"
+                    :color="value[key] ? '#2c82e0' : ''"
```

Comment:
Can't we move this color somwhere?
Also check other places. I see this color being used elsewhere.